src/Functions/custom/New-AzFunctionAppPlan.ps1

```diff
@@ -0,0 +1,175 @@
+        [System.String]
+        ${ResourceGroupName},
+
+        [Parameter(HelpMessage='The Azure subscription ID.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Runtime.DefaultInfo(Script='(Get-AzContext).Subscription.Id')]
+        [System.String]
+        ${SubscriptionId},
+
+        [Parameter(Mandatory=$true, HelpMessage='The plan sku. Valid inputs are: EP1, P2, EP3')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Body')]
+        [ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.Functions.Support.SkuType])]
+        [System.String]
+        # Sku (EP1, EP2 or EP3)
+        ${Sku},
+
+        [Parameter(Mandatory=$true, HelpMessage='The worker type for the plan. Valid inputs are: Windows or Linux.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Runtime')]
+        [ArgumentCompleter([Microsoft.Azure.PowerShell.Cmdlets.Functions.Support.WorkerType])]
+        [System.String]
+        # Worker type (Linux or Windows)
+        ${WorkerType},
+
+        [Parameter(Mandatory=$true, HelpMessage='The location for the consumption plan.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Path')]
+        [ValidateNotNullOrEmpty()]
+        [System.String]
+        ${Location},
+
+        [Parameter(HelpMessage='The maximum number of workers for the app service plan.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Body')]
+        [System.Int32]
+        [ValidateRange(0,20)]
+        [Alias("MaxBurst")]
+        ${MaximumWorkerCount},
+
+        [Parameter(HelpMessage='The minimum number of workers for the app service plan.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Body')]
+        [System.Int32]
+        [Alias("MinInstances")]
+        [ValidateRange(0,20)]
+        ${MinimumWorkerCount},
+
+        [Parameter(HelpMessage='Resource tags.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Runtime.Info(PossibleTypes=([Microsoft.Azure.PowerShell.Cmdlets.Functions.Models.Api20150801Preview.IResourceTags]))]
+        [System.Collections.Hashtable]
+        ${Tag},  
+
+        [Parameter(HelpMessage='Run the command as a job.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        ${AsJob},
+
+        [Parameter(HelpMessage='Run the command asynchronously.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        ${NoWait},
+
+        [Parameter()]
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.Functions.Category('Azure')]
+        [System.Management.Automation.PSObject]
+        ${DefaultProfile}
```

Comment:
same comments on 'DontShow' and 'AsJob' parameters

Reply:
As Above, for the AsJob issue, need to fix or remove the parameter for this release

Reply:
AsJob and NoWait parameters are commented out for now.